apps/src/templates/instructions/InstructionsWithWorkspace.jsx

```diff
@@ -2,7 +2,7 @@ import $ from 'jquery';
 import React, {PropTypes} from 'react';
 import {connect} from 'react-redux';
 var CodeWorkspaceContainer = require('../CodeWorkspaceContainer');
-var TopInstructions = require('./TopInstructions');
+import TopInstructions from './TopInstructions';
```

Comment:
Fixes a subtle problem - by adding a non-default export to the TopInstructions component for testing, the value loaded by a non-ES6 `require()` pattern changed from:

```
var TopInstructions = require('TopInstructions');
// Set to function (wrapped TopInstructions)
```

to

```
var TopInstructions = require('TopInstructions');
// Set to object {
//   default: function (wrapped TopInstructions),
//   UnconnectedTopInstructions: function UnconnectedTopInstructions
// }
```

Using an ES6 import resolves the problem by explicitly loading the default export, which was the original behavior.